downloadApolloSchema.sh

```diff
@@ -0,0 +1,9 @@
+./gradlew downloadApolloSchema --endpoint="https://rickandmortyapi.com/graphql" --schema="sample/src/main/graphql/com/chuckerteam/chucker/sample/schema.json"
+
+# GraphQL Schema Update
+# Open Android Studio terminal
+# Run command `./downloadApolloSchema.sh`
+# Format the file (`ctrl+option+i` && `option+cmd+L`), if need
+# Replace `"null"` to `null`, if need
```

Comment:
Why this is needed? How can a user know if they need to do it?

Reply:
the `if need` here was related to if we have these fields. Indeed could make confusing. Will remove.